robot_smach_states/src/robot_smach_states/util/designators/core.py

```diff
@@ -316,6 +338,55 @@ def test_list(self):
         self.assertEqual(v.resolve_type, [str])
         self.assertListEqual(v.resolve(), ['a', 'b', 'c'])
 
+    def test_none_value(self):
```

Comment:
These test are moved to the test folder in `rwc2019_robot_smach_states`, this PR should hold till that PR is merged and then these changes should be moved to the test_folder as well.